### PR TITLE
Climbable localization fixes

### LINKED ISSUE #3374: Vault message appears incorrectly
## Description
This is almost certainly because of the localization changes and almost certainly affects other things that I haven't noticed

**Screenshots**

image
You jump onto table (527, table) -- doesn't say 'the table', and shows the entityid and prototype name for some reason

**Additional context**

### PR DESCRIPTION
**Pull request by Macoron**
## About the PR 
Fixes #3374

**Screenshots**

**Changelog**

:cl:
- fix: Fix text messages for climbable objects (like tables or windows)

### CODE REVIEW

**Review by mirrorcult — COMMENTED** [review]
format should be `comp-climbable` rather than `climbable-component` for the localization strings (I think, check the other component localizations in case I'm insane)

**Comment by Macoron** [comment]
> format should be `comp-climbable` rather than `climbable-component` for the localization strings (I think, check the other component localizations in case I'm insane)

I copied naming style from `flash-component`. But `gas-tank-component` uses `comp-gas-tank`.
So which one is correct? 🤔

**Comment by mirrorcult** [comment]
@RemieRichards

**Comment by AJCM-git** [comment]
`flash-component` is wrong, the current convention is to use `comp-flash` (same for climbable)

**Comment by RemieRichards** [comment]
Yep. We settled on `comp-` after the first PR, it's been the majority format since.

**Comment by Macoron** [comment]
Moved all ids to `com-climbable`

File: Resources/Locale/en-US/components/climbable-component.ftl [hunk 1]
```diff
@@ -2,27 +2,27 @@
...
-climbable-component-cant-reach = You can't reach there!
+comp-climbable-cant-reach = You can't reach there!
 
 # Shown to you when your character can't interact with climbable for some reason
-climbable-component-cant-interact = You can't do that!
+comp-climbable-cant-interact = You can't do that!
 
 # Shown to you when your character can't climb
-climbable-component-cant-climb = You are incapable of climbing!"
+comp-climbable-cant-climb = You are incapable of climbing!"
```
Comment (mirrorcult) [inline Resources/Locale/en-US/components/climbable-component.ftl:28]: leftover quotation mark

**Pull request merged**